cktap/__init__.py

```diff
@@ -2,13 +2,4 @@
 # (c) Copyright 2021 by Coinkite Inc. This file is covered by license found in COPYING-CC.
 #
 
-__all__ = [ 'proto', 'exceptions', 'transport', 'constants', 'utils', 'sweep' ]
-
-from .version import __version__
-
-# find connected cards
-from .transport import find_cards, find_first
-
-# base class for working with cards, wants a transport
-from .proto import CKTapCard
```

Comment:
undo this breaking change

Reply:
I removed this as it was not working... If you just `clone` repo and try `pip install --editable .` It will go there from setup.py importing `find_cards` and `find_first` from transport and then it hit this line:
```
import sys, os, cbor2
```
where it fails as `cbor2` is not installed yet

Also why is it a breaking change ? this kind of import path `from cktap import ...` seems to be un-used, besides the README.md

Reply:
- we don't know how third-parties are using the library, so this may break code we cannot see out there
- and yet, it's early days and I don't think they would cry
- I have no idea what best practise here is w.r.t empty __init__.py files vs. version numbers and so on

Reply:
maybe like [this](https://github.com/coinkite/coinkite-tap-proto/pull/14/commits/c22d3aeb27299e82b82598974f7475fcde59c1b1) ?

Reply:
but I would rather have it hard-coded in `setup.py` if we have no use for it in cktap --> version of library should be clear from output of various `pip` commands

Reply:
- no a text file doesn't help. might as well be value in setup.py then
- my concern is customers may want to know what version of the library they have installed (at runtime)
- I've seen `package.__version__` in many other places

Reply:
* only __version__ is in `cktap.__init__.py`
* added command `cktap --version` to get version of library in runtime
* updated `README.md`

Reply:
![Screenshot from 2022-04-13 21-05-10](https://user-images.githubusercontent.com/25349625/163252923-750be202-c46f-4d3f-a215-0a272c04f226.png)